app/controllers/users.py

```diff
@@ -79,6 +81,18 @@ def post(self):
             template,
             subject
         )
+        
+        # check if user exists in anonymous table and assign them to a project
+        anonymous_user_exists = AnonymousUser.find_first(email=email)
+
+        if anonymous_user_exists:
+            project = Project.get_by_id(anonymous_user_exists.project_id)
+            user_details = User.find_first(email=email)
+            
+            new_role = ProjectUser(role=anonymous_user_exists.role, user_id=user_details.id)
+            project.users.append(new_role)
+            saved_project_user = project.save()
```

Comment:
I think a check to confirm the save should be added i.e **if not saved_project_user**: